### PR TITLE
chore: publish v1.0.0-dev.1 and update readme with new syntax

### PR DESCRIPTION
**Pull request by dshukertjr**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,28 @@
...
+  print('$error \n $stracktrace');
+}
+```
+- `count` and `head` can be specified within `FetchOptions()` in `.select()`
+```dart
+final response = await client.from('countries').select('*', FetchOptions(count: CountOption.exact));
+print(response.data);
+print(response.count);
+```
+- DEPRECATED: `.execute()` is not deprecated
```
Comment (Vinzent03) [inline CHANGELOG.md]: replace `not` with `now`

**Review by Vinzent03 — CHANGES_REQUESTED** [review]
Despite the typo, LGTM. After merging, I would migrate the dart client.

File: CHANGELOG.md (hunk 1 above)
Comment (dshukertjr) [reply CHANGELOG.md]: Thank you for this amazing catch!

Comment (dshukertjr) [reply CHANGELOG.md]: Fixed!

**Review by bdlukaa — APPROVED** [review]

**Pull request merged**